pkgs/development/compilers/abcl/update.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix-update curl
+
+new_version=$(curl https://armedbear.common-lisp.dev/ | grep abcl-src | sed 's;[^>]*>abcl-src-\(.*\).tar[^$]*;\1;' | head -n 1)
```

Comment:
How about `svn ls https://abcl.org/svn/tags | tail -1 | tr -d /` in combination with `nix-shell -i bash -p nix-update subversion` above ?

Reply:
Nice

Much smaller and simpler